### DISCUSSION TITLE
阿里副總裁王剛離職 曾掌舵AI、自動駕駛研發

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#267] [閒聊八卦]
此篇文章表示，阿里巴巴企業的副總裁王剛離職，其任職期間對AT開發以及無人駕駛技術有卓越貢獻，除此之外，我們也能從此文章中看見大陸對於AI技術有相當深入的開發及研究。
[文章連結](https://udn-com.cdn.ampproject.org/v/s/udn.com/news/amp/story/7333/6011471?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16413703261610&referrer=https%3A%2F%2Fwww.google.com&amp_tf=%E4%BE%86%E6%BA%90%EF%BC%9A%251%24s&ampshare=https%3A%2F%2Fudn.com%2Fnews%2Fstory%2F7333%2F6011471)

**Comment by zixunn** [comment]
自動駕駛一直是我很感興趣的科技新聞類別。
